app/assets/stylesheets/styles/components/pages/_how_it_works.css.scss

```diff
@@ -1,18 +1,20 @@
+// @import "compass/css3/flexbox";
+
 .how-it-works {
   background-color: #dcccb3;
-  height: 70%;
+  height: $hero-height-small;
   max-width: initial;
   overflow: hidden;
   position: absolute;
   // Initially position the how-it-works section above the screen
-  top: -200%;
+  top: -100%;
```

Comment:
Is there a reason this got set to -100? I think it is causing this to happen on smaller screens:

![screenshot 2015-04-20 15 30 22](https://cloud.githubusercontent.com/assets/485583/7226335/3e47aed6-e772-11e4-8ef0-364cd3b9804d.png)